azure/components/computecluster/setup.ftl

```diff
@@ -130,8 +130,34 @@
 
             [#case DB_COMPONENT_TYPE]
 
-                [#local dbSecrets = getSettingSecrets(linkTargetSettings, "ENV") +
-                    [{ "DB_PASSWORD" : linkTargetAttributes["SECRET"] }] ]
+                [#local dbSecrets = []]
+
+                [#switch linkTargetConfiguration.Solution.Engine]
```

Comment:
Should this be part of the dB component attributes? Would you expect the same to be applied for functions or other components that use dB connections

Reply:
Yes, good point. I'll implement that on the Azure provider now and push up the changes.

Reply:
@roleyfoley  I have updated the PR with commit 8eefc17 to include secret support on the component. I have written it in such a way that we can add Secret Store support at a later time and migrate it to the shared component with minimal effort.